uint64 timestamp
std_msgs/Float32MultiArray semantics
